programs/uxd/src/mercurial_utils/calculate_possible_lp_token_precision_loss_collateral_value.rs

```diff
@@ -0,0 +1,37 @@
+use crate::error::UxdError;
+use anchor_lang::{
+    prelude::{Account, Clock, SolanaSysvar},
+    Result,
+};
+use fixed::types::I80F48;
+use mercurial_vault::state::Vault;
+
+// Calculate how much collateral could be lost in possible LP token precision loss
+pub fn calculate_possible_lp_token_precision_loss_collateral_value(
```

Comment:
nit: for some reason this function is duplicated now: mercurial.rs and calculate_possible_lp_token_precision_loss_collateral_value.rs